### PR TITLE
AshampooGmbH&Co.KG.AshampooBurningStudioFREE version 1.24.12

### PR DESCRIPTION
**Pull request by palladinoandrea**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/108611)

### CODE REVIEW

**Comment by palladinoandrea** [comment]
@microsoft-github-policy-service agree

File: manifests/a/AshampooGmbH&Co/KG/AshampooBurningStudioFREE/1.24.12/AshampooGmbH&Co.KG.AshampooBurningStudioFREE.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: AshampooGmbH&Co.KG.AshampooBurningStudioFREE
```
Comment (Trenly) [inline manifests/a/AshampooGmbH&Co/KG/AshampooBurningStudioFREE/1.24.12/AshampooGmbH&Co.KG.AshampooBurningStudioFREE.installer.yaml:4]:
```suggestion
PackageIdentifier: Ashampoo.BurningStudio.Free
```
After changing this you will also need to update the file path to be /manifests/a/Ashampoo/BurningStudio/Free

File: manifests/a/AshampooGmbH&Co/KG/AshampooBurningStudioFREE/1.24.12/AshampooGmbH&Co.KG.AshampooBurningStudioFREE.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: AshampooGmbH&Co.KG.AshampooBurningStudioFREE
```
Comment (Trenly) [inline manifests/a/AshampooGmbH&Co/KG/AshampooBurningStudioFREE/1.24.12/AshampooGmbH&Co.KG.AshampooBurningStudioFREE.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Ashampoo.BurningStudio.Free
```
After changing this you will also need to update the file path to be /manifests/a/Ashampoo/BurningStudio/Free

File: manifests/a/AshampooGmbH&Co/KG/AshampooBurningStudioFREE/1.24.12/AshampooGmbH&Co.KG.AshampooBurningStudioFREE.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: AshampooGmbH&Co.KG.AshampooBurningStudioFREE
```
Comment (Trenly) [inline manifests/a/AshampooGmbH&Co/KG/AshampooBurningStudioFREE/1.24.12/AshampooGmbH&Co.KG.AshampooBurningStudioFREE.yaml:4]:
```suggestion
PackageIdentifier: Ashampoo.BurningStudio.Free
```
After changing this you will also need to update the file path to be /manifests/a/Ashampoo/BurningStudio/Free

File: manifests/a/AshampooGmbH&Co/KG/AshampooBurningStudioFREE/1.24.12/AshampooGmbH&Co.KG.AshampooBurningStudioFREE.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: AshampooGmbH&Co.KG.AshampooBurningStudioFREE
+PackageVersion: 1.24.12
+PackageLocale: en-US
+Publisher: Ashampoo GmbH & Co. KG
+PackageName: Ashampoo Burning Studio FREE
+License: Free
+Copyright: Ashampoo GmbH & Co. KG
+ShortDescription: Ashampoo Burning Studio FREE Setup
```
Comment (Trenly) [inline manifests/a/AshampooGmbH&Co/KG/AshampooBurningStudioFREE/1.24.12/AshampooGmbH&Co.KG.AshampooBurningStudioFREE.locale.en-US.yaml:11]: Please provide a better description of what Burning Studio is used for

File: manifests/a/AshampooGmbH&Co/KG/AshampooBurningStudioFREE/1.24.12/AshampooGmbH&Co.KG.AshampooBurningStudioFREE.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: AshampooGmbH&Co.KG.AshampooBurningStudioFREE
+PackageVersion: 1.24.12
+PackageLocale: en-US
+Publisher: Ashampoo GmbH & Co. KG
+PackageName: Ashampoo Burning Studio FREE
+License: Free
```
Comment (Trenly) [inline manifests/a/AshampooGmbH&Co/KG/AshampooBurningStudioFREE/1.24.12/AshampooGmbH&Co.KG.AshampooBurningStudioFREE.locale.en-US.yaml:9]:
```suggestion
License: Freeware
```

File: manifests/a/AshampooGmbH&Co/KG/AshampooBurningStudioFREE/1.24.12/AshampooGmbH&Co.KG.AshampooBurningStudioFREE.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: AshampooGmbH&Co.KG.AshampooBurningStudioFREE
+PackageVersion: 1.24.12
+PackageLocale: en-US
```
Comment (Trenly) [inline manifests/a/AshampooGmbH&Co/KG/AshampooBurningStudioFREE/1.24.12/AshampooGmbH&Co.KG.AshampooBurningStudioFREE.locale.en-US.yaml:6]:
```suggestion
PackageLocale: en-US
PrivacyUrl: https://www.ashampoo.com/en-us/privacy
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
Close with reason: Stale pull request;

**Pull request closed without merging**